.yamato/com.unity.ml-agents-test.yml

```diff
@@ -24,6 +31,22 @@ test_platforms:
     flavor: b1.medium
 ---
 
+all_package_tests:
+  name: Run All Combinations of Editors/Platforms Tests
```

Comment:
nit: "Wait For All Combinations..."?

Reply:
it actually triggers the jobs, so it's more of an aggregate of all tests

Reply:
going to leave as is.